### ISSUE TITLE
Profile page for Member/Activists and KYC

### ISSUE THREAD

**Issue by lapin7** [rchain/Members#203]
There's a need for a clear sign up procedure and a self-service to update personal profiles.
The personal profile contains following data:
Email
First Name
Middle Name
Last Name
Discord ID
Github ID
Telegram ID
QQI ID
Address
City
State
Country
GPS-location 2 digit (00,00)
Personal folder at GDrive: RChain/Contracts/Contractors/[First Name] [Last Name]
URL to SoW
URL to TaxForm
URL to Invoices
Main Work Category
Skills
ETH-address
BTC-address
......

Define also storage, ownership and access policy

**Comment by ResonanceXX** [comment]
Please is this on the github platform?

**Comment by lapin7** [comment]
No, I think it should be on the main site https://rchain.coop somewhere.
You login to have access to your profile page. But this still has to be made!!!

**Comment by Ojimadu** [comment]
@patrick727 along with the developer should have access to the back-end of the [website](member.rchain.coop). These updates are necessary for updating personal information of the coop members would call patrick's attention to it.
@lapin7 If this data fields are updated on the website there will be need for members to update their information on the coop's website.

**Comment by lapin7** [comment]
I think this could go through a google form.

--
Cheers,
HJ

**Comment by as1ndu** [comment]
@lapin7  can't we just build a custom in-house social network for rchain members?

It will simplify creation and tracking of issues plus communication amongst members.

**Comment by lapin7** [comment]
I think 🤔 GitHub works good enough. But I’m always open to better solutions as long it doesn’t start an endless Tool discussion.

**Comment by Keaycee** [comment]
We've got Github, Discord and telegram. Github is good enough to discuss
issues. Having issues discussed in so many channels make it unprofessional
and lacking orderliness. I suggest we keep up with Github.

**Comment by as1ndu** [comment]
>"Having issues discussed in so many channels make it unprofessional
and lacking orderliness"

@Keaycee  yes that what I am talking about.

Ithink i have a solution, will get back.

**Comment by Keaycee** [comment]
@as1ndu  we may like to see what you've got, but note that we don't intend taking out our issues on Github to another platform.

**Comment by as1ndu** [comment]
👍

**Comment by ResonanceXX** [comment]
Github user interface fits perfectly with the structure RChain wants to implement. Very easy to navigate and sends mails in real time. All of this is good. Github must remain. It serves a good purpose. 

Sent from Yahoo Mail on Android 
 
  On Sat, 13 Jan 2018 at 21:29, Keaycee<notifications@github.com> wrote:   
@as1ndu we may like to see what you've got, but note that we don't intend taking out our issues on Github to another platform.

**Comment by Tonyprisca13** [comment]
@ keaycee u are right.

**Comment by dckc** [comment]
lots of overlap with #222; I wonder if they're worth tracking separately.

**Comment by dckc** [comment]
@ian-bloom noted a new feature we're likely to need soon

 - do I want to share contact info with coop partner orgs / companies?
